galaxy_ng/app/access_control/statements/insights.py

```diff
@@ -1,4 +1,87 @@
+# Pulp ansible has the collections api broken down into a bunch of separate viewsets
+# (ie: collection versions, collections, download, upload etc.) Galaxy NG expects the
+# functionality of these viewsets to all be roughly the same, so instead of duplicating
+# the statenents for these viewsets, they're stored here and reused on a bunch of different
+# policies.
+_collection_statements = [
+    {
+        "action": ["list", "retrieve"],
+        "principal": "authenticated",
+        "effect": "allow",
+        "condition": "has_rh_entitlements",
+    },
+    {
+        "action": "destroy",
+        "principal": "authenticated",
+        "effect": "allow",
+        "condition": [
+            "has_model_perms:ansible.delete_collection",
+            "has_rh_entitlements",
+        ],
+    },
+    {
+        "action": ["download"],
+        "principal": "authenticated",
+        "effect": "allow",
+        "condition": "has_rh_entitlements",
+    },
+    {
+        "action": "create",
+        "principal": "authenticated",
+        "effect": "allow",
+        "condition": ["can_create_collection", "has_rh_entitlements"],
+    },
+    {
+        "action": "update",
+        "principal": "authenticated",
+        "effect": "allow",
+        "condition": "can_update_collection",
```

Comment:
it seems we have been missing `has_rh_entitlements` on the update action, we should be able to add it